static/js/reducers/financial_aid.js

```diff
@@ -44,45 +44,53 @@ export type FetchError = {
 
 
 export type FinancialAidState = {
-  income?:      string,
-  currency?:    string,
-  checkBox?:    boolean,
-  fetchStatus?: string,
-  programId?:   number,
-  validation?:  FinancialAidValidation,
-  fetchError?:  FetchError,
+  income?:          string,
+  currency?:        string,
+  checkBox?:        boolean,
+  fetchAddStatus?:  string,
+  fetchSkipStatus?: string,
+  programId?:       number,
+  validation?:      FinancialAidValidation,
+  fetchError?:      FetchError,
 };
 
 export const financialAid = (state: FinancialAidState = INITIAL_FINANCIAL_AID_STATE, action: Action) => {
   switch (action.type) {
   case START_CALCULATOR_EDIT:
-    return { ...FINANCIAL_AID_EDIT, programId: action.payload };
+    return {
+      ...FINANCIAL_AID_EDIT,
+      programId: action.payload,
+      fetchAddStatus: state.fetchAddStatus,
+      fetchSkipStatus: state.fetchSkipStatus,
+    };
   case CLEAR_CALCULATOR_EDIT:
-    return FINANCIAL_AID_EDIT;
+    return {
+      ...FINANCIAL_AID_EDIT,
+      fetchAddStatus: state.fetchAddStatus,
+      fetchSkipStatus: state.fetchSkipStatus,
+    };
```

Comment:
Maybe, this might be out of scope for this PR but it doesn't seem to me like a user should be able to initiate a `skip` request if they have a 'normal' request in flight at the same time.